sdk/storage/storage-queue/BreakingChanges.md

```diff
@@ -1,6 +1,21 @@
 # Breaking Changes
 
-## 2019.11 12.0.0-preview.5
+## 2019.11 12.0.0-preview.6
```

Comment:
Won't it be 12.0.0? The GA release?

Reply:
We have a separate process to update the version numbers across all the libraries that we are shipping for GA. Until then, all our libraries will have the preview suffix

Reply:
Ok. need to update the changelog too then.

Reply:
Yes, we will be taking care of that